alibi_detect/cd/base.py

```diff
@@ -0,0 +1,215 @@
+from abc import abstractmethod
+from functools import partial
+import logging
+import numpy as np
+from typing import Callable, Dict, Optional, Tuple, Union
+from alibi_detect.base import BaseDetector, concept_drift_dict
+from alibi_detect.cd.preprocess import preprocess_drift
+from alibi_detect.cd.utils import update_reference
+from alibi_detect.utils.statstest import fdr
+
+logger = logging.getLogger(__name__)
+
+
+class BaseUnivariateDrift(BaseDetector):
```

Comment:
Unfortunately, because of inheriting from this class, the sphinx API docs don't have entries for the methods `predict` etc. for the classes that inherit from this. Suggest opening a new issue to resolve this separately.

Reply:
Sounds good.